src/browser/api/external_window.ts

```diff
@@ -541,7 +541,8 @@ async function subscribeToInjectionEvents(externalWindow: Shapes.ExternalWindow)
 /*
     Decides whether external window is valid (external window filtering)
 */
-export function isValidExternalWindow(nativeWindowInfo: Shapes.NativeWindowInfo, ignoreVisibility?: boolean) {
+export function isValidExternalWindow(rawNativeWindowInfo: Shapes.RawNativeWindowInfo, ignoreVisibility?: boolean) {
+  const nativeWindowInfo = parseRawNativeWindowInfo(rawNativeWindowInfo);
   const classNamesToIgnore = [
```

Comment:
maybe refer to those as blacklistedExternalWins or something like that? also, considering this list is probably going to get bigger, think about moving these string consts to their own file...

Reply:
1. blacklistedExternalWins: "blacklisted" sounds a bit harsh, these external windows have done nothing wrong :) i'd prefer to keep something along ignore, omit, irrelevant, etc...
2. I don't think (hope) this list won't get much bigger, at least for Win10. Maybe Win7 might add a few here. Once it starts getting out of control, I agree, we should move them elsewhere then.